server/src/main/java/org/opensearch/rest/NamedRoute.java

```diff
@@ -144,7 +144,8 @@ private NamedRoute(Builder builder) {
                 "Invalid route name specified. The route name may include the following characters"
                     + " 'a-z', 'A-Z', '0-9', ':', '/', '*', '_' and be less than "
                     + MAX_LENGTH_OF_ACTION_NAME
-                    + " characters"
+                    + " characters, "
+                    + builder.uniqueName
```

Comment:
What is this for?

Reply:
Actually, I'll pull this out in to a smaller PR, basically an error is printed and it isn't clear which route is the problem, so you've got to repro in a debugging or updating the error message :D

Reply:
Circling back, I didn't end up removing this but seems pretty harmless to include as is.